CommonMark/Parser/InlineMethods.cs

```diff
@@ -14,11 +14,12 @@ internal static class InlineMethods
         /// Initializes the array of delegates for inline parsing.
         /// </summary>
         /// <returns></returns>
-        internal static Func<Subject, Inline>[] InitializeParsers(CommonMarkSettings settings)
+        internal static Func<Subject, CommonMarkSettings, Inline>[] InitializeParsers(CommonMarkSettings settings)
```

Comment:
The settings object is referenced by `PostProcessInlineStack()`. I believe they are required in the delegates.